docs/release-notes/4288-version.txt

```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+**Fixes**
+
+-  Automatically add a "determined_version" key to the metadata of checkpoints created by any of the
```

Comment:
If determined_version is an actual variable, it should be within double-backticks instead of quotes.
Also, "... in release 0.18.0." instead of "... in the 0.18.0 release."